app/Config/Feature.php

```diff
@@ -24,4 +24,11 @@ class Feature extends BaseConfig
      * @var bool
      */
     public $multipleFilters = false;
+
+    /**
+     * Use improved new auto routing, instead of the traditional auto routing.
+     *
+     * @var bool
+     */
+    public $autoRoutesImproved = false;
```

Comment:
`public bool` should be fine since it is new property?

```suggestion
    public bool $autoRoutesImproved = false;
```

Reply:
Thanks!